src/lib/cms/client.ts

```diff
@@ -0,0 +1,64 @@
+import 'whatwg-fetch';
+import {IRequestResponseStatus} from './models';
+
+/**
+ * Send a request to the server in order to update the draft
+ * CMS data defined at the specified path
+ * @param updatedData - Data to update in the draft CMS data document
+ * @param path - Path where the data should be updated
+ */
+export function updateDraftCmsData(updatedData: any, path: string[]): Promise<any> {
+  return new Promise((resolve, reject) => {
```

Comment:
No need to create a new Promise here. Just return the one you get from fetch + your transformations